test/message/promise_fast_reject.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
I'm not sure I understand this test, wouldn't either an unhandled rejection on the `'One'` or on the `'Two'` be acceptable? What is it specifying? Do we really have a guarantee that two rejections in the same tick are ordered?

Also, it might be useful to get all other currently unhandled rejections logged.